flow/scripts/pdn.tcl

```diff
@@ -30,5 +30,10 @@ if {[info exist ::env(PDN_CFG)]} {
 if {![info exists standalone] || $standalone} {
   # write output
   write_def $::env(RESULTS_DIR)/2_6_floorplan_pdn.def
+
+  if {[info exists ::env(POST_PDN_TCL)] && [file exists $::env(POST_PDN_TCL)]} {
+    source $::env(POST_PDN_TCL)
+  }
+
```

Comment:
why add this after writing of the DEF? If this is to correct incorrect behavior (like the wrong via enclosure), wouldn't the better solution be to correct the behavior in pdngen?

Reply:
@louiic why do we need this post-processor?  Is this something that pdngen is fixing?

Reply:
This is a place holder for OR-1004 ticket. The via enclosure for generated via is incorrect; thus, creating DRC for each via used in the power grid.

Reply:
and OR-1090